### ISSUE TITLE
Use MongoDB's weighted search for product search

### ISSUE THREAD

**Issue by diogotcorreia** [livraria-papelaria-espaco/lpe-website#40]
This requires waiting on https://github.com/strapi/strapi/issues/5972

When implemented, sort search results by relevance instead of by last updated.

**Comment by diogotcorreia** [comment]
We're now using Melisearch for search, so this is no longer relevant
